analyzers/nodejs/nodejs_test.go

```diff
@@ -131,3 +131,22 @@ func TestUsingNodeModuleFallback(t *testing.T) {
 
 	// assert.NotEmpty(t, analysisResults.Direct)
 }
+
+func TestOnlyProdDeps(t *testing.T) {
+    m := module.Module{
+		Dir:  filepath.Join("testdata", "prodonly"),
+		Type: pkg.NodeJS,
+	}
+
+	a, err := analyzers.New(m)
+	assert.NoError(t, err)
+
+	a.(*nodejs.Analyzer).Tool = MockNPM{
+		JSONFilename: filepath.Join("testdata", "prodonly", "package.json"),
```

Comment:
This should actually be the route to a fixture with the output of `npm ls --json`, not to a manifest directly.